pulseapi/entries/views.py

```diff
@@ -208,11 +209,13 @@ class EntryView(RetrieveAPIView):
     queryset = Entry.objects.public()
     serializer_class = EntrySerializer
     pagination_class = None
+    parser_classes = (JSONParser,)
```

Comment:
As a tuple, just to make it explicit this really is one, it might be wise for us to split this over multiple lines.
```
parser_classes = (
  JSONParser,
)
```
(and then the same note for the next handful of same lines of course)